lms/templates/hera/not-found/not-found-ask.html

```diff
@@ -0,0 +1,12 @@
+<%page expression_filter="h"/>
+<%inherit file="/main.html" />
+
+<div class="not-found-holder">
+    <h1 class="not-found-title">
+        Please wait for your teacher before proceeding.
+        <br>
+    </h1>
+    <p class="not-found-text">
+        If you still have trouble, please email <a href="mailto:openedx@act.org" class="not-found-link" target="_blank">openedx@act.org</a>
```

Comment:
please use this variable to get email address `settings.DEFAULT_FROM_EMAIL`

Reply:
done